app/initializers/feature-flags.js

```diff
@@ -1,5 +1,6 @@
 export function initialize(application) {
-  // application.inject('controller', 'featureFlags', 'service:feature-flags');
+  application.inject('controller', 'featureFlags', 'service:feature-flags');
```

Comment:
I would inject this service all across the controllers and components; let's inject the service manually where it's needed.